### ISSUE TITLE
MACS2 controls

### ISSUE THREAD

**Issue by sklasfeld** [taoliu/MACS#264]
Hello,
 I was just wonder how MACS2 deals with unequal treatments and controls. For example, if I have three input replicates, does it make sense to use them all even if I only have treatment ChIP? For example, should I use `-t treatment1.bam -c control1.bam control2.bam control3.bam` OR should I try normalizing the three control files and averaging them? Similarly, if I use `-t treatment1.bam treatment2.bam -c control1.bam` will MACS2 account for the number of samples being different in the control and treatment?
Thank you so much for your time. I look forward to a response soon.
Sammy

**Comment by taoliu** [comment]
Hi Sammy, the default and recommended behavior in MACS2 is to "linearly scale larger sample down to the smaller sample". So if you have more reads, after filtering out duplicates, in control than in treatment, the signals in control will be scaled down. For example, if you have 3 million reads in treatment, and 5million reads in control, the control signals will all be multiplied with 3/5 (or 0.6). If you have more treatment reads, the treatment signals will be scaled down.

**Comment by fnhe** [comment]
did you find the best way to deal with this problem? I have 4treatmentA, 4treatmentB, but only got 2 control bam files

**Comment by sklasfeld** [comment]
Yeah I just ended up using all the control bam files and letting MACS2 do the random down sampling.
